lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -1739,6 +1792,27 @@ class EngineSemanticsOwner {
     });
   }
 
+  final Map<int, Tombstone> _tombstones = <int, Tombstone>{};
+
+  DomElement createElementFor(int nodeId) {
+    final Tombstone? tombstone = _tombstones.remove(nodeId);
+    if (tombstone != null) {
+      // The semantics node has returned, so the tombstone is no longer
+      // necessary. As I (yjbanov) was testing this (July 19, 2023), I did not
```

Comment:
Can this `createElementFor` happen without the `nodeId` tombstone being ever blurred (and thus scheduled for cleanup anyway)?